### DISCUSSION TITLE
`excludeWalletIds` Parameter Not Working, Target Wallet Still Displayed in Wallet List.

### DISCUSSION THREAD

**Discussion by hangts** [reown-com/appkit#2657] [Q&A]
**Description:**

We are using `web3modal` in our dApp and we want to exclude the Keplr Wallet from the wallet list. I have set the `excludeWalletIds` parameter, but it is not working as expected. The Keplr Wallet is still displayed in the wallet list.

**Current Technical Implementation:**

- **Framework**: `React`
- **Version**: `@web3modal/ethers5: 5.0.11`

**Configuration Initialization:**

```javascript
createWeb3Modal({
    ethersConfig: defaultConfig({
        metadata,
        enableEIP6963: true,
        enableInjected: false,
        enableCoinbase: false,
    }),
    defaultChain: chains[0],
    chains,
    projectId,
    enableAnalytics: true,
    featuredWalletIds:[
        'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96',
        '971e689d0a5be527bac79629b4ee9b925e82208e5168b733496a09c0faed0709',
    ],
    excludeWalletIds: [
        '6adb6082c909901b9e7189af3a4a0223102cd6f8d5c39e39f3d49acb92b578bb'
    ],
    allWallets: 'HIDE',
})
```

**Problem Description:**

Even though I have added the Keplr Wallet's ID (`6adb6082c909901b9e7189af3a4a0223102cd6f8d5c39e39f3d49acb92b578bb`) to the `excludeWalletIds` parameter, it still appears in the wallet list.

**Expected Solution:**

I would like to be able to correctly exclude the Keplr Wallet or be provided with an alternative solution to achieve this functionality.

**Additional Information:**

- If there is any further information required, please let me know.
- Thank you for the support and assistance!

**Comment by rtomas** [comment]
please, Can you upgrade to the latest version and try again ?
